bpf/flows.c

```diff
@@ -51,13 +51,27 @@
  */
 #include "network_events_monitoring.h"
 
+static inline void update_dns(additional_metrics *extra_metrics, pkt_info *pkt, int dns_errno) {
+    if (pkt->dns_id != 0) {
```

Comment:
u can check dns_err here and if its `0` then add DNS info else add dns_error

Reply:
there can be an error and still have DNS data: when the dns request wasn't found, we can't generate latency, but we still provide the other data, and return ENOENT.
So it shouldn't be a if/else here, if we want to still provide what we have